tests/dummy/app/templates/docs/index.md

```diff
@@ -8,9 +8,10 @@ ember install ember-caluma-form-builder
 
 `ember-caluma-form-builder` is a [routable engine](http://ember-engines.com).
 To include it in your application just mount it in your `app/router.js` and
-add some dependencies in `app/app.js`:
+add dependencies in `app/app.js` and modify the apollo service with a mixin:
```

Comment:
Yeah the docs are not really ready yet. I filed an issue for that: #18 

I think it's not really worth it to change this now since the process will change again anyways..